packages/aws-cdk/test/integ/cli/bootstrapping.integtest.ts

```diff
@@ -196,7 +196,10 @@ integTest('switch on termination protection, switch is left alone on re-bootstra
 
   await fixture.cdk(['bootstrap', '-v', '--toolkit-stack-name', bootstrapStackName,
     '--termination-protection', 'true',
-    '--qualifier', fixture.qualifier], { modEnv: { CDK_NEW_BOOTSTRAP: '1' } });
+    '--qualifier', fixture.qualifier,
+    '--cloudformation-execution-policies', 'arn:aws:iam::aws:policy/AdministratorAccess'], {
+    modEnv: { CDK_NEW_BOOTSTRAP: '1' },
+  });
   await fixture.cdk(['bootstrap', '-v', '--toolkit-stack-name', bootstrapStackName, '--force'], { modEnv: { CDK_NEW_BOOTSTRAP: '1' } });
```

Comment:
does this also need to be updated to include `--cloudformation-execution-policies` - assuming the test is intending to only switch on the `--termination-protection` flag

Reply:
No.